modules/win_mozilla_build_tester/manifests/pip.pp

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class win_mozilla_build_tester::pip {
+  if ($facts['custom_win_location'] == 'azure') and ($facts['custom_win_bootstrap_stage'] == 'complete') {
+    $cache_drive  = 'y:'
```

Comment:
This will be the variable we will want to pass to the pip conf template.

Reply:
Will update in RELOPS-719